runtime/filetype.vim

```diff
@@ -2259,6 +2259,9 @@ au BufNewFile,BufReadPost *.tssop		setf tssop
 " TSS - Command Line (temporary)
 au BufNewFile,BufReadPost *.tsscl		setf tsscl
 
+" Tutor mode
+au BufNewFile,BufReadPost *.tutor		setf tutor
```

Comment:
`.vimtutor` could avoid potential conflicts. Though it's probably very unlikely...